### ISSUE TITLE
Button: focused UX wrong

### ISSUE THREAD

**Issue by danyball** [axa-ch/patterns-library#2085]
Keyboard focus should have same effect like mouse hover.
see https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=225%3A240

- Button
- button-link

There should not be an outline:

![image](https://user-images.githubusercontent.com/7448004/103758656-c6ceb100-5012-11eb-9de5-6fa4db76827d.png)

**Comment by salle18** [comment]
I just saw this issue and strongly disagree with this, from a11y perspective keyboard focus outline is different thing from mouse hover. This should be readdressed with designers, I know they hate outline but it is there for a reason.

On another note I don't have access to that figma link so I can't really see what intended behavior is 😄

**Comment by danyball** [comment]
> I just saw this issue and strongly disagree with this, from a11y perspective keyboard focus outline is different thing from mouse hover. This should be readdressed with designers, I know they hate outline but it is there for a reason.
> 
> On another note I don't have access to that figma link so I can't really see what intended behavior is 😄

At the link you postet at the issue they wrote that it is ok that a element do not have an outline if it has another visual indication that the element is focuses. And our button has got this indication because the background turns into another color. If you do not like it please discuss that with the designers and create another issue here. Thanks!
